### ISSUE TITLE
DL3008 even if pin-version is set

### ISSUE THREAD

**Issue by eric0915** [hadolint/hadolint#685]
- [x] This is a bug report

### Expected behavior
No linting errors

### Actual behavior
For Haskell Dockerfile Linter v1.16.3-0-g6a8dd04: 

`Dockerfile:3 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>``

and 

For Haskell Dockerfile Linter 2.6.0-no-git:

`Dockerfile:3 DL3008 warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>``

### Steps to reproduce the behavior
`hadolint Dockerfile`

```Dockerfile
FROM ubuntu:bionic

RUN apt-get update -y && \
    apt-get install -y --no-install-recommends bind9utils=1:9.11.3+dfsg-1ubuntu1.15 \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

**Used in WSL2**

**Comment by m-ildefons** [comment]
Hi Eric,
you have an error in your shell syntax. There needs to be a `;` or a `&&` between the `apt-get install -y --no-install-recommends bind9utils=1:9.11.3+dfsg-1ubuntu1.15` and the `apt-get clean`.

The way the command is written right now, the shell parser thinks you want to install the packages `bind9utils=1:9...`, `apt-get` and `clean`, but if you separate the two commands, it will correctly recognize the `bind9utils=1:9...` as an argument to the `apt-get install` and the `apt-get clean` as a separate command.

Hope this helps.
